charts/ccsm-helm/values.yaml

```diff
@@ -99,6 +99,9 @@ zeebe:
     -XX:ErrorFile=/usr/local/zeebe/data/zeebe_error%p.log
     -XX:+ExitOnOutOfMemoryError
 
+  persistence:
```

Comment:
:x: In general I like that we have a separate section, ideally I would expect that we have the other configs underneath but this would be a breaking change. Maybe it makes more sense to just rename it to `pvcEnabled`